marklogic-rdf4j/src/test/java/com/marklogic/semantics/rdf4j/MarkLogicRepositoryConnectionTest.java

```diff
@@ -662,6 +660,9 @@ public void testHasStatementBNode() throws Exception
         Statement st1 = vf.createStatement(alice, name, alicesName);
         conn.add(st1, context1);
         Assert.assertTrue(conn.hasStatement(null, null, alicesName, false));
+
+        BNode markName = vf.createBNode();
+        Assert.assertFalse(conn.hasStatement(null, null, markName, false));
```

Comment:
Hi @grechaw , as quickly described here https://github.com/marklogic/marklogic-sesame/issues/364 we can just save the BNode id and create a "new" bnode with this saved ID.

So in the tests it would look something like:

    String marksBNodeId = "marksBnodeId"
    BNode markName = vf.createBNode(marksBNodeId);
    Assert.assertTrue(conn.hasStatement(null, null, markName, false));